### PR TITLE
Added fire command placeholder with 'pew!'

### LINKED ISSUE #3: Bleepy controls
- [x] Flap command
- [ ] Placeholder gun command
- [x] Automated linear decrease in bleepy's angle

Abstract controls-object to reuse with nemoustache!

### PR DESCRIPTION
**Pull request by alexwalterbos**
Closes #3 

Press `F` for fire!

@AlexKolpa  Learned a lot about c++ in this one! Please check the `makefile` changes carefully:

### CODE REVIEW

**Comment by alexwalterbos** [comment]
will update with master tomorrow

File: pew.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	glTexEnvf(GL_TEXTURE_2D, GL_TEXTURE_ENV_MODE, GL_MODULATE);
+	glDepthMask(GL_FALSE);
+
+	glBegin(GL_QUADS);
+	glTexCoord2f(0.f,1.f); glVertex3f(-1.f, 0.5f, 0.f);
+    	glTexCoord2f(1.f,1.f); glVertex3f( 1.f, 0.5f, 0.f);
+	glTexCoord2f(1.f,0.f); glVertex3f( 1.f, -0.5f, 0.f);
+	glTexCoord2f(0.f,0.f); glVertex3f(-1.f, -0.5f, 0.f);
+	glEnd();
+	glPopMatrix();
```
Comment (AlexKolpa) [inline pew.cpp]: No need for this `glPopMatrix`. I just realized the same goes for the call in `bird.cpp`.

**Comment by AlexKolpa** [comment]
You should reset the pew when the game resets.

**Comment by alexwalterbos** [comment]
@AlexKolpa ready for review. I've also tweaked the controls a bit.

**Comment by AlexKolpa** [comment]
:+1:

**Pull request merged**